bokeh/colors/util.py

```diff
@@ -52,8 +52,6 @@ class _ColorGroupMeta(type):
     ''' This metaclass enables ColorGroup class types to be used like simple
     enumerations.
 
-    When Python2 support is dropped, this will no longer be necessary.
```

Comment:
I'm not actually sure what we would use besides a metaclass, since these methods can't be added to `type` (e.g. by a class decorator instead)

Reply:
I guess we could use a module-getattr to returned new *instances* (rather than classes) that had this same API that is currently bolted on to the class type. Will leave that for some other time.